ESLint-Formatter.py

```diff
@@ -143,11 +143,37 @@ def get_node_path():
     return node
 
   @staticmethod
+  def get_local_eslint(root, folders):
+    pkg = os.path.join(root, 'package.json')
+    esl = 'node_modules/.bin/eslint'
+
+    # find package.json first
+    if os.path.isfile(pkg):
+      # get eslint from local node_modules
+      return os.path.join(root, esl)
```

Comment:
I believe that if you have a package.json, but do not have eslint installed locally, this would return the path as if you did have it installed, which then wouldn't be caught by the None check on 172.

I think there needs to be a check here to ensure that `os.path.join(root, esl)` is a file